### PR TITLE
feat: update pnpm

### PR DESCRIPTION
**Pull request by helciofranco**
- Closes https://github.com/FuelLabs/fuel-connectors/issues/81

---

Update `pnpm` to `v9.5.0`

### CODE REVIEW

**Review by arthurgeron — APPROVED** [review]

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
we should merge it when upgrading the other repos as well

`fuels-wallet` `fuel-explorer` `fuels-npm-packs`

**Comment by helciofranco** [comment]
alright, I will hold this one @LuizAsFight

**Review by arthurgeron — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -48,5 +48,5 @@
       "fast-xml-parser@<4.4.1": ">=4.4.1"
     }
   },
-  "packageManager": "pnpm@8.15.7"
+  "packageManager": "pnpm@9.5.0"
```
Comment (arthurgeron) [inline package.json:51]: I think we can remove this and allow toolings to use their latest PNPM 9 minor/patch versions

File: e2e-tests/package.json [hunk 2]
```diff
@@ -20,9 +20,9 @@
   },
   "engines": {
     "node": ">=18",
-    "pnpm": ">=8"
+    "pnpm": ">=9"
   },
-  "packageManager": "pnpm@8.15.7",
+  "packageManager": "pnpm@9.5.0",
```
Comment (arthurgeron) [inline e2e-tests/package.json:25]: I think we can remove this and allow toolings to use their latest PNPM 9 minor/patch versions

**Review by arthurgeron — DISMISSED** [review]

File: e2e-tests/package.json (hunk 2 above)
Comment (helciofranco) [reply e2e-tests/package.json]: yes, makes sense
thanks

https://github.com/FuelLabs/fuel-connectors/pull/224/commits/948eab1e0590c0a10097d0028ca9a6ab7e020ddb

**Review by arthurgeron — APPROVED** [review]

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**